rosie.py

```diff
@@ -1,49 +1,64 @@
-from sys import argv, exit
+"""Rosie.
 
+Usage:
+  rosie.py run (chamber_of_deputies | federal_senate)
+  rosie.py run (chamber_of_deputies | federal_senate) [--years <years>]
+  rosie.py run (chamber_of_deputies | federal_senate) [--path <path>]
+  rosie.py run (chamber_of_deputies | federal_senate) [--path] [--years <years>]
+  rosie.py test [chamber_of_deputies | federal_senate]
 
-def entered_command(argv):
-    if len(argv) >= 2:
-        return argv[1]
-    return None
+Options:
+    --years   runs Rosie fetching data and finding suspicions for a specific set of years
+    --path    directory in which Rosie saves fetched data and the suspicions CSV [default: /tmp/serenata-data]
 
+"""
 
-def help():
-    message = (
-        'Usage:',
-        '  python rosie.py run chamber_of_deputies [<path to output directory>]',
-        'Testing:',
-        '  python rosie.py test',
-        '  python rosie.py test chamber_of_deputies',
-    )
-    print('\n'.join(message))
+from sys import exit
+from docopt import docopt
 
 
-def run():
+def main():
+    arguments = docopt(__doc__)
+
+    if arguments['run']:
+        run(**arguments)
+
+    if arguments['test']:
+        return test(**arguments)
+
+
+def run(**args):
     import rosie
     import rosie.chamber_of_deputies
     import rosie.federal_senate
 
-    if len(argv) >= 3:
-        target_module = argv[2]
-    else:
-        print('A module must be provided.')
-        help()
-        exit(1)
-    target_directory = argv[3] if len(argv) >= 4 else '/tmp/serenata-data/'
+    target_directory = args['<path>'] if args['--path'] else '/tmp/serenata-data/'
+
+    target_module = 'chamber_of_deputies' if args['chamber_of_deputies'] else 'federal_senate'
```

Comment:
@cuducos I'm not into this way of selecting `target_module` do you have an better option for that?

Reply:
When using `docopt` we should have in `arguments` a dictionary like this:

```python
{'--path': False,
 '--years': False,
 'chamber_of_deputies': True,
 'core': False,
 'federal_senate': False,
 'run': True,
 'test': False}
```

So I'd go for something like:

```python
commands = ('chamber_of_deputies', 'federal_senate')
target_module, *_ =  filter(lambda x: arguments.get(x), commands)
```